inst/shiny/modules/column_mapping.R

```diff
@@ -268,10 +268,16 @@ column_mapping_server <- function(id, data, manual_units, on_submit) {
       }
 
       # Reorder columns based on the desired order
-      dataset %>%
+      dataset <- dataset %>%
         relocate(all_of(desired_order)) %>%
-        mutate(TIME = ifelse(DOSNO == 1, AFRLT, ARRLT)) %>% #TODO: Remove this after AUC0 merged
-        processed_data()
+        mutate(TIME = ifelse(DOSNO == 1, AFRLT, ARRLT))#TODO: Remove this after AUC0 merged
+
+      # Apply labels to the dataset
+      message("Calling apply_labels function")
```

Comment:
issue: Please use `log_trace` instead of `message`. Furthermore, I would argue it is not worth keeping this log anyway.